src/main/kotlin/bakalarka/elearningplatform/model/Course.kt

```diff
@@ -0,0 +1,41 @@
+package bakalarka.elearningplatform.model
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import org.springframework.stereotype.Component
+import javax.persistence.*
+
+@Component
+@Entity
+@Table(name = "COURSE")
+data class Course(
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    var id: Long? = null,
+    var title: String,
+    @Column(length = 15000)
+    var description: String,
+    var shortDescription: String,
+    var thumbnail: String,
+    var fee: Long,
+    var language: String,
+    var topic: TOPIC,
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "instructor_id")
+    @JsonIgnoreProperties("course")
+    val instructor: Instructor?,
```

Comment:
Musi to byt nullable? Nie som si 100% isty, ale bolo by cool, keby nemusi :-D Ale asi aj do `JoinColumn` by malo ist `nullable = false`. Skus, ci to tak pojde.

Pri dalsich uz iba napisem NULLABLE.